frontend-react/src/shared/HeroWrapper/HeroWrapper.tsx

```diff
@@ -0,0 +1,19 @@
+import classNames from "classnames";
+
+import styles from "./HeroWrapper.module.scss";
+
+export interface HeroWrapperProps
+    extends React.PropsWithChildren<React.HTMLAttributes<HTMLDivElement>> {
+    isAlternate?: boolean;
+}
+
+export function HeroWrapper({ isAlternate, ...props }: HeroWrapperProps) {
+    const classnames = classNames(
```

Comment:
I don't think we should be passing down any css classes. If we have styles / functionality that are toggled on or off, they should be passed down as props.

Reply:
These classes are passed down as props.

Reply:
Right, but instead can we just create a prop within the `HeroWrapper` that then turns on / off styling instead of needing to manually pass in classes?